test/services/implementations/broker.test.js

```diff
@@ -135,24 +136,18 @@ describe('Test: Internal broker', function () {
         return new WSServerMock();
       };
 
-      client = new WSBrokerClient('internalBroker', {}, kuzzle.pluginsManager);
+      client = new WSBrokerClient('internalBroker', kuzzle.config.internalBroker, kuzzle.pluginsManager);
       client.ws = () => new WSClientMock(server.server);
 
-      return q.all([
```

Comment:
We delay server and client initialization here because we need some tests to be run before the server initialization.